presto-main/src/main/java/io/prestosql/spiller/FileSingleStreamSpiller.java

```diff
@@ -150,6 +155,7 @@ private void writePages(Iterator<Page> pageIterator)
             }
         }
         catch (UncheckedIOException | IOException e) {
+            this.filesystemErrorHandler.run();
```

Comment:
- there is no point to use `this.`
- should we wrap this with try-catch so we don't swallow the current exception? If so please extract the method.

Reply:
Fixed usage of `this`, where applicable.

We could wrap the `fileSystemErrorHandler` in an exception, but the current impl is `LoadingCache::invalidateAll`, which doesn't throw an exception.